nms/app/packages/magmalte/app/views/equipment/GatewayDetailStatus.js

```diff
@@ -30,6 +39,20 @@ export default function GatewayDetailStatus({gwInfo}: {gwInfo: lte_gateway}) {
     checkInTime = new Date(gwInfo.status.checkin_time);
   }
 
+  const startTime = Math.floor(Date.now() / 1000);
+  const {isLoading: _isLoading} = useMagmaAPI(
+    MagmaV1API.getNetworksByNetworkIdPrometheusQueryRange,
+    {
+      networkId: networkId,
+      query: `cpu_percent{gatewayID=~"${gwInfo.id}", service="magmad"}`,
```

Comment:
Hey Hanna, Why do we have gatewayID=~"${gwInfo.id}" and why gatewayID="${gwInfo.id}" ?